build.gradle

```
@@ -509,6 +509,23 @@ task runProductionAutoTestServer(type: JavaExec, dependsOn: [remapJar, remapTest
 	}
 }
 
+def addPomMetadataInformation(MavenPom pom) {
+	pom.licenses {
+		license {
+			name = "Apache 2.0"
```

```suggestion
			name = "Apache-2.0"
```

According to the [docs](https://maven.apache.org/pom.html#Licenses) Using an [SPDX identifier](https://spdx.org/licenses/) as the license name is recommended.